lib/ipmi_strings.c

```diff
@@ -830,15 +830,15 @@ int ipmi_oem_info_init()
 	if ((home = getenv("HOME"))) {
 		char path[PATH_MAX];
 		strncpy(path, home, sizeof(path));
-		strcat(path, "/.local" IANA_PEN_REGISTRY);
+		strcat(path, "/" IANAUSERDIR "/" IANA_PEN_REGISTRY);
```

Comment:
similarly, is there not a safe strcat?

Reply:
Yes, should probably use `strncat()` instead. Or did you mean anything else?

Reply:
That's what I meant, thanks! (I hadn't looked it up just assumed there was an "n" variation).

Reply:
There is also `snprintf()` and some people say it is safer than `strncat()` but I don't see why.
Nonetheless, I'll probably use `snprintf()`. Hopefully that will make Codacy CI plugin happy.